Microsoft.Toolkit.Win32/Microsoft.Windows.Interop.WindowsXamlHost.WinForms/DesktopWindowXamlSourceExtensions.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Toolkit.Win32.UI.Interop
+{
+    /// <summary>
+    /// COM wrapper required to access native-only methods on DesktopWindowXamlSource
+    /// </summary>
+    public static class DesktopWindowXamlSourceExtensions
+    {
+        /// <summary>
+        /// Gets the IDesktopWindowXamlSourceNative interface from a DesktopWindowXamlSource instance. This
+        /// interface is the only way to set DesktopWindowXamlSource's target window for rendering.
+        /// </summary>
+        /// <param name="desktopWindowXamlSource">The DesktopWindowXamlSource instance to get the interface from</param>
+        /// <returns>IDesktopWindowXamlSourceNative interface pointer</returns>
+        public static IDesktopWindowXamlSourceNative GetInterop(this Windows.UI.Xaml.Hosting.DesktopWindowXamlSource desktopWindowXamlSource)
```

Comment:
This should be marked `SecurityCritical` since it calls a security critical method